src/components/calendar/calendar.css

```diff
@@ -37,6 +38,19 @@
   padding-top: var(--spacing-medium);
 }
 
+.calendar-year-list {
+  height: calc(var(--spacing-x-large) * 7); /* 7 list options */
+  padding-left: var(--spacing-medium);
+  margin-top: calc(0 - var(--spacing-medium));
```

Comment:
I don't understand the margin setup. What's the intention behind negative margin? The other margin?